src/stub/src/arch/powerpc/64le/bxx.S

```diff
@@ -37,7 +37,7 @@ ppcbxx:  # (*f_unf)(xo->buf, out_len, h.b_cto8, h.b_ftid);
 #define ptr0 a4
 
         cmplwi cr0,ftid,0xd0;       bnelr- cr0  # if (0xd0!=ftid)   return;
-        rlwinm. len,len,32-2,2,31; beqlr- cr0  # if (0==(len>>=2)) return;
+        rldic. len,len,2,1; beqlr- cr0  # if (0==(len>>=2)) return;
```

Comment:
That gives the right register contents, but remember to record the condition ==0 for the beqlr- which follows.